man/systemd.xml

```diff
@@ -270,9 +270,8 @@
     ideas behind systemd, please refer to the
     <ulink url="http://0pointer.de/blog/projects/systemd.html">Original Design Document</ulink>.</para>
 
-    <para>Note that some but not all interfaces provided
-    by systemd are covered by the
-    <ulink url="Portability and">Interface Portability and Stability Promise</ulink>.</para>
+    <para>Note that some but not all interfaces provided by systemd are covered by the
+    <ulink url="https://systemd.io/PORTABILITY_AND_STABILITY/">Interface Portability and Stability Promise</ulink>.</para>
```

Comment:
the trailing / is so weird... i wonder if we should remove that, even if the web server immediately redirects to it again